event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateHandlerTest.java

```diff
@@ -592,11 +519,119 @@ void shouldHandleJournalWithMultipleTypes() throws IOException {
+                                                                         BigDecimal totalPoints) {
+        return CandidateEvaluatedMessage.builder()
+                   .withCandidateType(createExpectedCandidate(instanceType,
+                                                              Map.of(CRISTIN_NVI_ORG_TOP_LEVEL_ID,
+                                                                     points.setScale(SCALE, RoundingMode.HALF_UP)),
+                                                              publicationChannel, Level.LEVEL_ONE.getValue(),
+                                                              basePoints, totalPoints, bucketUri))
+                   .build();
+    }
+
+    private static NviCandidate createExpectedCandidate(InstanceType instanceType,
+                                                        Map<URI, BigDecimal> institutionPoints,
+                                                        PublicationChannel channelType, String level,
+                                                        BigDecimal basePoints, BigDecimal totalPoints,
+                                                        URI publicationBucketUri) {
+        var verifiedCreators = List.of(new NviCreator(HARDCODED_CREATOR_ID, List.of(CRISTIN_NVI_ORG_SUB_UNIT_ID)));
+        return NviCandidate.builder()
+                   .withPublicationId(HARDCODED_PUBLICATION_ID)
+                   .withPublicationBucketUri(publicationBucketUri)
+                   .withDate(HARDCODED_PUBLICATION_DATE)
+                   .withInstanceType(instanceType)
+                   .withChannelType(channelType.getValue())
+                   .withLevel(level)
+                   .withPublicationChannelId(HARDCODED_PUBLICATION_CHANNEL_ID)
+                   .withIsInternationalCollaboration(false)
+                   .withCollaborationFactor(ONE.setScale(1, ROUNDING_MODE))
+                   .withCreatorShareCount(countCreatorShares(verifiedCreators))
+                   .withBasePoints(basePoints)
+                   .withNviCreators(verifiedCreators)
+                   .withInstitutionPoints(institutionPoints.entrySet().stream()
+                                              .map(entry -> new InstitutionPoints(entry.getKey(), entry.getValue(),
+                                                                                  List.of(
+                                                                                      new CreatorAffiliationPoints(
+                                                                                          HARDCODED_CREATOR_ID,
+                                                                                          CRISTIN_NVI_ORG_SUB_UNIT_ID,
+                                                                                          entry.getValue()))))
+                                              .toList())
+                   .withTotalPoints(totalPoints)
```

Comment:
meh

Reply:
Can you be more specific? 😄 Add builder for `InstitutionPoints` and `CreatorAffiliationPoints`?